OpenEXR/IlmImf/ImfDwaCompressor.cpp

```diff
@@ -1010,7 +1010,7 @@ DwaCompressor::LossyDctDecoderBase::execute ()
 
             for (int y = 8 * blocky; y < 8 * blocky + maxY; ++y)
             {
-                if ((size_t)_rowPtrs[comp][y] & _SSE_ALIGNMENT_MASK)
+                if (reinterpret_cast<size_t>(_rowPtrs[comp][y]) & _SSE_ALIGNMENT_MASK)
```

Comment:
a static_cast to uintptr_t would be idiomatic here

Reply:
Done